packages/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -35,6 +35,7 @@ class GoogleMap extends StatefulWidget {
     this.minMaxZoomPreference = MinMaxZoomPreference.unbounded,
     this.rotateGesturesEnabled = true,
     this.scrollGesturesEnabled = true,
+    this.zoomControlsEnabled = false,
```

Comment:
Shouldn't it be true by default, to match the behavior in the Android SDK.

Reply:
This has been adjusted. But I'm not a big fan of having the default value being written in Dart and Android.